ivy/functional/ivy/nest.py

```diff
@@ -1160,12 +1160,14 @@ def nested_map(
     to_ignore = ivy.default(to_ignore, ())
     extra_nest_types = ivy.default(extra_nest_types, ())
     if include_derived is True:
-        include_derived = {tuple: True, list: True, dict: True}
+        include_derived = {str(tuple): True, str(list): True, str(dict): True}
```

Comment:
we could replace `str(tuple)` by `"tuple"` here and so on.

Reply:
I'll start working on it